app.py

```diff
@@ -180,6 +180,8 @@
         "SERVER_PORT": "8086",
         "SPRING_CLOUD_CONFIG_URI": "http://openchallenges-config-server:8090",
         "SERVICE_REGISTRY_URL": "http://openchallenges-service-registry:8081/eureka",
+        "OPENCHALLENGES_IMAGE_SERVICE_THUMBOR_HOST": "https://dev.openchallenges.io/img/",
```

Comment:
This is not compatible with the way we want to deploy when we will merge to the stage and prod branch.

Reply:
@zaro0508 One solution would be to create three different parameter in the SSM parameters stores for the three deployments. For example:

- `openchallenges/dev/openchallenges-image-service-thumbor-host`
- `openchallenges/stage/openchallenges-image-service-thumbor-host`
- `openchallenges/prod/openchallenges-image-service-thumbor-host`

Then parametrize the name of parameter so that it's a function of the deployment name (dev, stage, prod).

Reply:
you can just use the `{fully_qualified_domain_name}` variable.  Take a look at the example in the [oc_app_props ServiceProps object](https://github.com/Sage-Bionetworks-IT/openchallenges/blob/dev/app.py#L290-L304)